### ISSUE TITLE
Move Code-Based Speed Limits to DM

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#2285]
Waiting on further discussion about streets data.

**Comment by alan-deanda** [comment]
The data continues to live in MAINT. It might be a good idea to revisit this once new speed limits are established in the near future.

**Comment by alan-deanda** [comment]
Closing as this is no longer needed with the new speed limit data being worked on by Vision Zero and TED.
